frontend/appflowy_flutter/lib/plugins/database/board/application/board_bloc.dart

```diff
@@ -273,6 +273,11 @@ class BoardBloc extends Bloc<BoardEvent, BoardState> {
               );
             }
           },
+          openRowDetail: (rowMeta) {
+            final copyState = state;
+            emit(BoardState.openRowDetail(rowMeta: rowMeta));
+            emit(copyState);
```

Comment:
The openRowDetail state is a temporary state that will trigger the bloc listeners, but not build any widgets because it doesn't contain the required, relevant data.